src/bindings/python/src/openvino/frontend/onnx/__init__.py

```diff
@@ -8,10 +8,6 @@
 
 # flake8: noqa
 
-from openvino.utils import _add_openvino_libs_to_search_path
-
-_add_openvino_libs_to_search_path()
```

Comment:
now (after "namespaces" were merged), we can remove this `_add_openvino_libs_to_search_path` call even in files with pattern `openvino/XXX/__init__.py`